contracts/oraclize/MarketContractOraclize.sol

```diff
@@ -30,19 +29,12 @@ contract MarketContractOraclize is MarketContract, usingOraclize {
     // constants
     string public ORACLE_DATA_SOURCE;
     string public ORACLE_QUERY;
-    uint public ORACLE_QUERY_REPEAT;
     uint constant public QUERY_CALLBACK_GAS = 180000;
```

Comment:
I assume we can lower this now that the code path is much simpler.

Reply:
Yes, we can lower it, tests say ~115k, coverage network a bit more, so 130000 - 140000 should be a safe value.